graylog2-web-interface/src/components/common/SearchForm.jsx

```diff
@@ -54,6 +62,10 @@ class SearchForm extends React.Component {
   };
 
   static defaultProps = {
+    query: '',
+    onQueryChange: null,
```

Comment:
I would use `() => {}` as default prop, in that way we don't need to check if the function is `null` or not.